topics/thelounge/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: The Lounge
+github_url: https://github.com/thelounge/lounge
+logo: thelounge.png
+related: thelounge-theme, irc, irc-client, chat, javascript
```

Comment:
Theme should probably be an alias.

Reply:
I hesitated with this, but wasn't sure it was appropriate here. But thinking back about it, I'm convinced it's the right thing actually. Will change.

Reply:
Done.